lib/lotus/helpers/html_helper.rb

```diff
@@ -184,8 +193,13 @@ module HtmlHelper
       #
       # @see Lotus::Helpers::HtmlHelper
       # @see Lotus::Helpers::HtmlHelper::HtmlBuilder
-      def html
-        HtmlBuilder.new
+      # @see Lotus::Helpers::HtmlHelper::HtmlFragment
+      def html(&blk)
+        if block_given?
+          HtmlBuilder.new.fragment(&blk)
```

Comment:
@huynhquancam Good point about `FormBuilder` :+1: Never mind, please leave the initializer as it is.

Only one thing about `#html`. For performance reasons is it possible to avoid to bind `&blk` for each call? In other words:

```ruby
def html
  builder = HtmlBuilder.new
  builder.fragment { yield } if block_given?
  builder
end
```